assets/src/components/GradeDistribution.js

```diff
@@ -1,9 +0,0 @@
-import createHistogram from './d3/createHistogram'
-import withResponsiveness from './withResponsiveness'
-import createChartComponent from './createChartComponent'
-import compose from '../util/compose'
-
-export default compose(
-  withResponsiveness,
-  createChartComponent
-)(createHistogram)
```

Comment:
I thought this file is being used for Grades view?

Reply:
The file name is misleading, it isn't being used. 

The dependencies for Grade Distribution view are:
```javascript
import React from 'react'
import { renderToString } from 'react-dom/server'
import useFetch from '../hooks/useFetch'
import { withStyles } from '@material-ui/core/styles'
import Paper from '@material-ui/core/Paper'
import Grid from '@material-ui/core/Grid'
import Typography from '@material-ui/core/Typography'
import Histogram from '../components/Histogram'
import Spinner from '../components/Spinner'
import createToolTip from '../util/createToolTip'
import Table from '../components/Table'
import { average } from '../util/math'
```